test-data/unit/check-abstract.test

```diff
@@ -871,6 +871,17 @@ class A(metaclass=ABCMeta):
     def x(self, x): pass
 [out]
 
+[case testMixinTypedAbstractProperty]
+from abc import ABCMeta, abstractproperty
+class A(metaclass=ABCMeta):
+    @abstractproperty
+    def foo(cls) -> str:
+        pass
+class Mixin:
+    foo = "foo"
+class C(Mixin, A):
+    pass
+[out]
```

Comment:
I would add couple more tests with a non-abstract property that check:
* the error is still shown if you switch the order of bases in `C`.
* the error is not shown if type in `Mixin` is a strict subtype of type in property (it is safe since it is read-only)

Reply:
Added some more tests.